index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Ingrese su nombre");
+alert(`Hola ${nombreUsuario}`);
```

Comment:
En este caso me parecería mejor que no uses interpolación de string

```suggestion
alert(`Hola ` + nombreUsuario);
```